src/Domain/CoordinateMaps/Wedge2D.hpp

```diff
@@ -0,0 +1,69 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+/// \file
+/// Defines the class Wedge2D.
+
+#pragma once
+
+#include <array>
+#include <memory>
+#include <pup.h>
+
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "Domain/Direction.hpp"
+
+namespace CoordinateMaps {
+
+/// \ingroup CoordinateMaps
+/// Two dimensional map from the unit square to a wedge, which forms one
+/// quadrant of an annulus. The wedge can be oriented in the +/- x or y axis,
+/// with the inner and outer radii specifiable.
+class Wedge2D {
+ public:
+  static constexpr size_t dim = 2;
+
+  explicit Wedge2D(double inner_radius, double outer_radius,
+                   Direction<2> positioning_of_wedge);
+
+  Wedge2D() = default;
+  ~Wedge2D() = default;
+  Wedge2D(Wedge2D&&) = default;
+  Wedge2D& operator=(Wedge2D&&) = default;
+  Wedge2D(const Wedge2D&) = default;
+  Wedge2D& operator=(const Wedge2D&) = default;
+
+  template <typename T>
+  std::array<std::decay_t<tt::remove_reference_wrapper_t<T>>, dim> operator()(
+      const std::array<T, dim>& x) const noexcept;
+
+  // Currently unimplemented, hence the noreturn attribute
+  template <typename T>
+  [[noreturn]] std::array<std::decay_t<tt::remove_reference_wrapper_t<T>>, dim>
+  inverse(const std::array<T, dim>& x) const noexcept;
+
+  template <typename T>
+  Tensor<std::decay_t<tt::remove_reference_wrapper_t<T>>,
+         tmpl::integral_list<std::int32_t, 2, 1>,
+         index_list<SpatialIndex<2, UpLo::Up, Frame::NoFrame>,
```

Comment:
can this 2 be replaced with dim

Reply:
Why? This is only 2D.

Reply:
There are other places where dim is used instead of 2 (line 50), so I was wondering if there is a choice being made here to use dim vs 2

Reply:
Consistency is probably good :)